integrationtests/integrationtests_suite_test.go

```diff
@@ -78,6 +78,9 @@ func init() {
 }
 
 var _ = BeforeEach(func() {
+	// set custom time format for logs
+	utils.SetLogTimeFormat("15:04:05.000")
```

Comment:
I still think it's kind of ridiculous have code in a QUIC library just for this, but if you really must…